src/bin/pgcopydb/pgsql.c

```diff
@@ -5627,3 +5627,28 @@ parseSentinel(void *ctx, PGresult *result)
 
 	context->parsedOK = true;
 }
+
+/*
+ * escape_identifier escapes the (postgres) identifiers and the generated string
+ * is always quoted. It uses PQescapeIdentifier
+ * (https://www.postgresql.org/docs/current/libpq-exec.html#LIBPQ-PQESCAPEIDENTIFIER).
+ */
+char *
+escape_identifier(PGSQL *pgsql, char *src)
```

Comment:
Please follow the per-module C nomenclature that we use here, which means naming that function `pgsql_escape_identifier`. Also add a note in the function comment about the memory allocation here, and how to free it. I believe `PQfreemem` should be used, right?